pvlib/soiling.py

```diff
@@ -76,8 +76,17 @@ def hsu(rainfall, cleaning_threshold, tilt, pm2_5, pm10,
     # cleaning is True for intervals with rainfall greater than threshold
     cleaning_times = accum_rain.index[accum_rain >= cleaning_threshold]
 
-    horiz_mass_rate = pm2_5 * depo_veloc['2_5']\
-        + np.maximum(pm10 - pm2_5, 0.) * depo_veloc['10'] * 3600
+    # determine the time intervals in seconds (dt_sec)
+    dt = rainfall.index
+    dt2 = np.roll(dt.copy(), 1)
```

Comment:
is copy needed?

Reply:
this line and the line below equivalent to `dt3 = (dt[1:] - dt[:-1]).total_seconds()`?